### PR TITLE
TAI-3746 Bring error format in line with GDS

### PR DESCRIPTION
**Pull request by adamliptrot-oc**

### CODE REVIEW

File: app/views/includes/forms/errorSummary.scala.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<div class="error-summary@if(form.hasErrors) { error-summary--show} @classes.mkString(" ")"
+    id="error-summary-display"
+    role="alert"
+    aria-labelledby="error-summary-heading"
+    tabindex="-1">
+    <h2 id="error-summary-heading" class="heading-medium error-summary-heading">@heading</h2>
+    <ul class="error-summary-list">
+    @if(form.hasErrors) {
+        @form.errors.map { error =>
+            <li role="tooltip" @dataJourney.map(page => Html(s"""data-journey="${page}:error:${error.key}""""))>
```
Comment (markhunter27) [inline app/views/includes/forms/errorSummary.scala.html]: I need to look into the error detail with regards to stageprompt GA.
Paul Cook pointed out that recent changes (probably radio changes) have broken our GA with regard to setting the 'error - field' event category on a failed submit. I haven't had a chance to poke around yet though.
Looks like we don't get the data-journey attribute set at the moment (on the WDYWTD page at least)

File: app/views/includes/forms/textArea.scala.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+@*
```
Comment (markhunter27) [inline app/views/includes/forms/textArea.scala.html]: Will this be a headstart on 3721?

File: conf/messages [hunk 3]
```diff
@@ -61,6 +61,7 @@ tai.returnToIncomeTaxSummary=Return to your Income Tax Summary
 tai.phoneNumber=Phone number
 tai.whatDoYouWantToTell=What do you want to tell us?
 tai.whatDoYouWantToDo=What do you want to do?
+tai.inPounds = in pounds.
```
Comment (markhunter27) [inline conf/messages]: picky - space on left of equals :)

File: app/views/includes/forms/textArea.scala.html (hunk 2 above)
Comment (adamliptrot-oc) [reply app/views/includes/forms/textArea.scala.html]: Yes

File: app/views/includes/forms/errorSummary.scala.html (hunk 1 above)
Comment (adamliptrot-oc) [reply app/views/includes/forms/errorSummary.scala.html]: Yeah, sounds like a separate ticket

**Pull request merged**